app/src/components/modal/modal_your_connection/index.tsx

```diff
@@ -103,18 +104,30 @@ const AccountInfoAddress = styled.p`
   font-size: ${props => props.theme.fonts.defaultSize};
   color: ${props => props.theme.colors.textColorDark};
   margin: 0;
+  width: 78px;
+  line-height: ${props => props.theme.fonts.defaultLineHeight}
+  letter-spacing: 0.2px;
+  height: 16px;
```

Comment:
@gem1no Imma kill you man! No heights or widths :D let the size of content determine that we just set the margins and paddings in most cases....